build.gradle

```diff
@@ -672,6 +672,7 @@ project(':streams') {
   archivesBaseName = "kafka-streams"
 
   dependencies {
+    compile project(':core')
```

Comment:
@enothereska and @ijuma is right. We are trying deliberately to not include `core` as a dependency of `streams`.

One example is for the Zkclient-related module for auto-creating topics, before `KIP-4` was introduced. We intentionally duplicate some of the code from `core` in order to not introducing the dependency.

I'm OK with putting it under the kafka tools package, it should be fine.